newIDE/app/src/EventsSheet/ParameterRenderingService.js

```diff
@@ -60,7 +72,10 @@ export default {
     if (components.hasOwnProperty(fieldType)) return components[fieldType];
     else return components.default;
   },
-  renderParameterString: (type: string, value: string) => {
+  renderParameterString: (type: string, value: string, renderObjectThumnail=null) => {
+    if (renderObjectThumnail) {
```

Comment:
renderObjectThumnail should always be passed and never be null.
Flow will help you if you had the proper typing: `(string) => React.Node` (make sure to import React properly for this: `import * as React from 'react'`). Also do this for InstructionsList and all components that are needed.

Reply:
Thank you for this. I wasn't sure what to put for flow there